### PR TITLE
jan ide release note

### PR DESCRIPTION
**Pull request by mirnawong1**
release note for jan ide updates. 

see release note: https://www.notion.so/dbtlabs/Update-IDE-release-notes-for-Jan-2023-6a0d67e417e04bc4a0fdb825b5c43eb8

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Unable to close toast notifications for command failure when the history drawer is open
+- Users ran into `Something went wrong` when previewing a model
+- Unable to retrieve repository status in the IDE due to inability to find SSH folder
+- Scroll bars and download CSV flicker within the preview pane
+
+## Coming soon 
+
+- dbt Labs will roll out a series of features that should improve the quality of life within the IDE over the next several months, such as autosave, the ability to revert individual files, and user experience improvements, like right-clicking.
+
+- dbt Labs is researching ways to include Linting/SQL Fluff in the IDE. If anyone is interested in sharing how you're using SQL Fluff today, please reach out to dbt Labs Product Manager, [Nate Nunta](bossman's email or dm?).
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]: hey @bossnunta - if users want to reach out to you, would it only be via slack dm's or email or another option? thinking of users that may not have slack

```suggestion
- dbt Labs is researching ways to include Linting/SQL Fluff in the IDE. If anyone is interested in sharing how you're using SQL Fluff today, please reach out to dbt Labs Product Manager, [Nate Nunta](bossman's email or dm?).
```

Comment (bossnunta) [reply website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]: I think let's leave my name in there but -- DM me on community Slack or use this email cloud-ide-feedback@dbtlabs.com

**Review by bossnunta — DISMISSED** [review]
Other than the one comment. This looks great to me!

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Cloud IDE updates and fixes"
+id: "ide-updates"
+description: "Cloud IDE enhancements"
+sidebar_label: "IDE updates and fixes"
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]: I'm not married to this wording, but I wonder if we should keep consistent with previous naming conventions? I update [this section](https://www.notion.so/dbtlabs/Request-a-dbt-Cloud-release-note-f962f39b49af4a23811d70984fef4f65#bf538d5b1cb44ba5ba58fb0411d80783) of our internal release notes page to capture these differences.  
```suggestion
sidebar_label: "Update and Fixes: IDE"
```

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- Performance improvements and reduced IDE slowness. The IDE should feel faster and snappier.
+- Reliability improvements – Improved error handling that previously put IDE in a bad state
+- Corrected the list of dropdown options for the Build button
+- Adjusted startup page duration
+- Added code snippets for `unique` and `not_null` tests for yml files
+- Added code snippets for metrics based on environment dbt versions
+- Changed “commit and push” to “commit and sync” to better reflect the action
+- Improved error message when saving or renaming files to duplicate names
+
+## Bugs fixed
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]: Just changing this to parallel other sections
```suggestion
## Bug fixes
```

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- Corrected the list of dropdown options for the Build button
+- Adjusted startup page duration
+- Added code snippets for `unique` and `not_null` tests for yml files
+- Added code snippets for metrics based on environment dbt versions
+- Changed “commit and push” to “commit and sync” to better reflect the action
+- Improved error message when saving or renaming files to duplicate names
+
+## Bugs fixed 
+
+- Users running into `RPC server got an unknown async ID` arbitrarily
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]:
```suggestion
- You no longer arbitrarily encounter an `RPC server got an unknown async ID` message
```

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+- The build button dropdown was hidden behind the placeholder DAG screen
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]:
```suggestion
- You can now see the build button dropdown, which had been hidden behind the placeholder DAG screen
```

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+- Unable to close toast notifications for command failure when the history drawer is open
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]:
```suggestion
- You can now close toast notifications for command failure when the history drawer is open
```

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+- Users ran into `Something went wrong` when previewing a model
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]:
```suggestion
- You no longer encounter a `Something went wrong` message when previewing a model
```

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+- Unable to retrieve repository status in the IDE due to inability to find SSH folder
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]: Was there an error message this addressed?
```suggestion
- You can now see repository status in the IDE, and the IDE finds the SSH folder
```

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+- Scroll bars and download CSV flicker within the preview pane
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]:
```suggestion
- Scroll bars and download CSV no longer flicker within the preview pane. 
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
Hey @mirnawong1 I added a few minor requests. I rewrote the fixed bugs to state the fix rather than the original problem, but feel free to change that wording.

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]: oh good callout, thank you Leona! i should've noticed that 🙏

**Comment by mirnawong1** [comment]
hey @runleonarun suggestions have been committed, thank you for those knowledge bombs!

File: website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+- Improved error message when saving or renaming files to duplicate names
+
+## Bug fixes
+
+- You no longer arbitrarily encounter an `RPC server got an unknown async ID` message
+- You can now see the build button dropdown, which had been hidden behind the placeholder DAG screen
+- You can now close toast notifications for command failure when the history drawer is open
+- You no longer encounter a `Something went wrong` message when previewing a model
+- You can now see repository status in the IDE, and the IDE finds the SSH folder
+- Scroll bars and download CSV no longer flicker within the preview pane.
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/01-Jan-2023/ide-updates.md]:
```suggestion
- Scroll bars and download CSV no longer flicker within the preview pane
```

**Review by runleonarun — APPROVED** [review]
LGTM!

**Pull request merged**